zebra-rpc/src/methods/tests/prop.rs

```diff
@@ -334,16 +334,51 @@ proptest! {
             );
 
             let call_task = tokio::spawn(rpc.get_raw_mempool());
-            let mut expected_response: Vec<String> = transaction_ids
-                .iter()
-                .map(|id| id.mined_id().encode_hex())
-                .collect();
-            expected_response.sort();
 
-            mempool
-                .expect_request(mempool::Request::TransactionIds)
-                .await?
-                .respond(mempool::Response::TransactionIds(transaction_ids));
+
+            #[cfg(not(feature = "getblocktemplate-rpcs"))]
```

Comment:
Looks like there are some unused imports and incorrect variable names in this file?

I think the imports need to be inside ` #[cfg(not(feature = "getblocktemplate-rpcs"))]`.

Reply:
That's strange, it's used here too: https://github.com/ZcashFoundation/zebra/pull/5994/files#diff-e5495906830cf674b84fb812a0cc57fa47692019f4aa7becd00c033a067ada9aR372